Makefile

```diff
@@ -19,7 +19,7 @@ GATEWAY_LINUX_BINARY = $(GATEWAY_BINARY_NAME)-linux
 
 OUTPUT_DIR = bin
 
-VERSION_PACKAGE = github.com/radu-matei/kube-toolkit/pkg/version
+VERSION_PACKAGE = $(shell scripts/gopkg)/pkg/version
```

Comment:
`scripts/gopkg` prints `github.com/radu-matei/kube-toolkit` when `make` is run from within `$GOPATH/src/github.com/radu-matei/kube-toolkit` for example :)

This makes Makefile universal. It is universal because I can now just build a kube-toolkit project(by forking this repository for now, but by running `ktk init` in the future) by just running `make`. No need to modify `VERSION_PACKAGE`.